core/src/test/java/de/danoeh/antennapod/core/storage/DbReaderTest.java

```diff
@@ -31,6 +38,7 @@
  */
 @SuppressWarnings("ConstantConditions")
 @RunWith(RobolectricTestRunner.class)
+@Config(sdk = {Build.VERSION_CODES.O_MR1})
```

Comment:
Why? Looks like you need to update your local Java install instead (I think API30 is only supported on Java 9+)